client/src/app/panel/tabs/linting/__tests__/LintingTabSpec.js

```diff
@@ -55,8 +55,10 @@ describe('<LintingTab>', function() {
           name: 'Foo',
           message: 'Foo message',
           rule: 'foo-rule',
-          documentation: {
-            url: 'https://foo.bar'
+          meta: {
```

Comment:
I believe this requires adjustment in `@camunda/linting` to ensure that [custom URLs are appropriately injected](https://github.com/search?q=repo%3Acamunda%2Flinting%20getDocumentationUrl&type=code):

Reply:
Thinking about it, https://github.com/camunda/linting/pull/129 should be the way to go. We can simply inject documentation in the [original compat rules](https://github.com/camunda/bpmnlint-plugin-camunda-compat) and this PR is fine without additional adjustments. :clap